scripts/develop

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+const Watchpack = require('watchpack');
+const notifier = require('node-notifier');
+
+const config = require('./lib/config');
+const eslint = require('./lib/eslint');
+const {flowStatus} = require('./lib/flow');
+const {mochaUnit, mochaFunctional} = require('./lib/mocha');
+
+console.log('Starting flow server...');
+if (!flowStatus()) {
+  process.exit(1);
+}
+
+const wp = new Watchpack();
+wp.watch(config.watch.files, config.watch.dirs);
+
+function notify(message) {
+  notifier.notify({title: 'web-ext develop: ', message});
+}
+
+let changed = new Set();
+
+wp.on('change', (changedFile) => {
+  changed.add(changedFile);
+});
+
+wp.on('aggregated', () => {
+  // Filter out temp files from detected changes.
+  const changes = Array.from(changed).filter((filePath) => {
+    return !path.basename(filePath).startsWith('.#');
```

Comment:
What is so special about files that start with `.#`? There are also many other kinds of temp files (e.g. ending with `~`), so I'm a bit curious here.

Reply:
The temp files created by my usual editor (emacs) starts with `.#`, and so this part was an unfinished (as the actual goal was to filter out files that may be changed whereas we may don't want them to trigger a run of all these tasks.

As agreed during our last triage meeting, I changed it to filter out any file that starts with a dot instead.